### PR TITLE
updates

### PR DESCRIPTION
**Pull request by matzew**
latest of UPS and improvements (e.g. WF-9)

### CODE REVIEW

File: wildfly/Dockerfile [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
 
 # Run everything below as root user
 USER root
 
-# Set the Java environment variable
-ENV JAVA_HOME /usr/lib/jvm/java-1.7.0-openjdk
-
 # Clean the metadata
-RUN yum -q clean all
+RUN yum install -y unzip && yum install -y wget && yum -q clean all
```
Comment (goldmann) [inline wildfly/Dockerfile]: Why two `yum` calls here? Install wget and unzip in sigle call for better build performance :)

Comment (matzew) [inline wildfly/Dockerfile]: oh, like

```
RUN yum install -y unzip wget
```

? if so, let me fix tht on ours

**Comment by goldmann** [comment]
@matzew Added you to the contributors, feel free to push directly, open/merge PR on your own :)

**Comment by matzew** [comment]
OPEN THE GATES ! :)

On Thu, Dec 3, 2015 at 10:29 AM, Marek Goldmann <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> Added you to the contributors, feel
> free to push directly, open/merge PR on your own :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-dockerfiles/aerogear/pull/4#issuecomment-161565660>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Pull request merged**